src/style.css

```diff
@@ -2,0 +1,29 @@
+.page {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+  margin: 0;
```

Comment:
Be consistent with your margins. Add only top or bottom margins, but not both to avoid potential margin collapse. Here, `.stars` has a `margin: 0;` declaration which overrides both top and bottom margins. If you need to reset the margin, do it in a more targeted fashion.